### DISCUSSION TITLE
Chakra Avatar with Next/Image

### DISCUSSION THREAD

**Discussion by dkrefta** [chakra-ui/chakra-ui#7512] [Q&A]
hey everyone, all good?

I was checking the lighthouse to see the performance and the tool is asking for the next/image on the Chakra Avatar images, there's a way to handle that, or only if I create an avatar component with the Next/Image

Thanks!!

**Comment by ghost** [comment]
Now I am facing a similar problem.
The Avatar component of ChakraUI provides a very useful default avatar, etc., so you'll need to use Next/Image in conjunction with this.

Using the Link component with Next/Link was not available with Avatar.
When I set Next/Image for the as property, all the elements disappeared
